Author: Cesare Falco <c.falco@ubuntu.com>
Description: Fix and bump release in manpages.
 Fix a couple of typos and bump release in manpages.
Forwarded: yes

Index: mame/src/osd/sdl/man/castool.1
===================================================================
--- mame.orig/src/osd/sdl/man/castool.1	2014-08-15 05:24:23.000000000 +0200
+++ mame/src/osd/sdl/man/castool.1	2014-10-16 15:19:11.851521000 +0200
@@ -6,7 +6,7 @@
 .\" Cesare Falco <c.falco@ubuntu.com>, February 2011
 .\"
 .\"
-.TH CASTOOL 1 2014-07-31 0.154 "MESS Generic cassette manipulation tool"
+.TH CASTOOL 1 2014-10-15 0.155 "MESS Generic cassette manipulation tool"
 .\"
 .\"
 .\" NAME chapter
Index: mame/src/osd/sdl/man/chdman.1
===================================================================
--- mame.orig/src/osd/sdl/man/chdman.1	2014-08-15 05:24:23.000000000 +0200
+++ mame/src/osd/sdl/man/chdman.1	2014-10-16 15:19:29.991521000 +0200
@@ -6,7 +6,7 @@
 .\" Ashley T. Howes <debiandev@ashleyhowes.com>, February 2005
 .\" updated by Cesare Falco <c.falco@ubuntu.com>, February 2007
 .\"
-.TH CHDMAN 1 2014-07-31 0.154 "MAME Compressed Hunks of Data (CHD) manager"
+.TH CHDMAN 1 2014-10-15 0.155 "MAME Compressed Hunks of Data (CHD) manager"
 .\"
 .\" NAME chapter
 .SH NAME
Index: mame/src/osd/sdl/man/imgtool.1
===================================================================
--- mame.orig/src/osd/sdl/man/imgtool.1	2014-08-15 05:24:23.000000000 +0200
+++ mame/src/osd/sdl/man/imgtool.1	2014-10-16 15:19:53.147521000 +0200
@@ -6,7 +6,7 @@
 .\" Cesare Falco <c.falco@ubuntu.com>, February 2011
 .\"
 .\"
-.TH IMGTOOL 1 2014-07-31 0.154 "MESS media image manipulation tool"
+.TH IMGTOOL 1 2014-10-15 0.155 "MESS media image manipulation tool"
 .\"
 .\"
 .\" NAME chapter
Index: mame/src/osd/sdl/man/jedutil.1
===================================================================
--- mame.orig/src/osd/sdl/man/jedutil.1	2014-08-15 05:24:23.000000000 +0200
+++ mame/src/osd/sdl/man/jedutil.1	2014-10-16 15:20:20.379521000 +0200
@@ -8,7 +8,7 @@
 .\" References
 .\" http://aarongiles.com/?p=159
 .\"
-.TH JEDUTIL 1 2014-07-31 0.154 "MAME JEDEC file utilities"
+.TH JEDUTIL 1 2014-10-15 0.155 "MAME JEDEC file utilities"
 .\"
 .\" NAME chapter
 .SH NAME
Index: mame/src/osd/sdl/man/ldresample.1
===================================================================
--- mame.orig/src/osd/sdl/man/ldresample.1	2014-08-15 05:24:23.000000000 +0200
+++ mame/src/osd/sdl/man/ldresample.1	2014-10-16 15:20:37.131521000 +0200
@@ -3,7 +3,7 @@
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
 .\"
-.TH LDRESAMPLE 1 2014-07-31 0.154 "MAME laserdisc audio manipulation tool"
+.TH LDRESAMPLE 1 2014-10-15 0.155 "MAME laserdisc audio manipulation tool"
 .\"
 .\" Please adjust this date whenever revising the manpage.
 .\"
Index: mame/src/osd/sdl/man/ldverify.1
===================================================================
--- mame.orig/src/osd/sdl/man/ldverify.1	2014-08-15 05:24:23.000000000 +0200
+++ mame/src/osd/sdl/man/ldverify.1	2014-10-16 15:20:55.243521000 +0200
@@ -5,7 +5,7 @@
 .\" Man page created from source and usage information by 
 .\" Cesare Falco <c.falco@ubuntu.com>, August 2008
 .\"
-.TH LDVERIFY 1 2014-07-31 0.154 "MAME laserdisc data checker"
+.TH LDVERIFY 1 2014-10-15 0.155 "MAME laserdisc data checker"
 .\"
 .\" NAME chapter
 .SH NAME
Index: mame/src/osd/sdl/man/mame.6
===================================================================
--- mame.orig/src/osd/sdl/man/mame.6	2014-08-15 05:24:23.000000000 +0200
+++ mame/src/osd/sdl/man/mame.6	2014-10-16 15:18:28.987521000 +0200
@@ -13,7 +13,7 @@
 .\" and updated by Andrew Burton <burtona@gol.com>, July 2003
 .\"
 .\"
-.TH MAME 6 2014-07-31 0.154 "MAME \- The Multiple Arcade Machine Emulator"
+.TH MAME 6 2014-10-15 0.155 "MAME \- The Multiple Arcade Machine Emulator"
 .\"
 .\"
 .\" NAME chapter
@@ -933,8 +933,8 @@
 .B \-natural, \-nat
 Allows user to specify whether or not to use a natural keyboard.
 This allows you to start your game or system in a 'native' mode, depending
-on your region, allowing compatability for non\-"QWERTY" style keyboards.
-The default is OFF (-nonatural).
+on your region, allowing compatibility for non\-"QWERTY" style keyboards.
+The default is OFF (\-nonatural).
 .TP
 .B \-joystick_contradictory
 Enable contradictory direction digital joystick input at the same time.
Index: mame/src/osd/sdl/man/mess.6
===================================================================
--- mame.orig/src/osd/sdl/man/mess.6	2014-08-15 05:24:23.000000000 +0200
+++ mame/src/osd/sdl/man/mess.6	2014-10-16 15:18:50.503521000 +0200
@@ -16,7 +16,7 @@
 .\" http://www.mess.org/
 .\"
 .\"
-.TH MESS 6 2014-07-31 0.154 "The Multiple Emulator Super System (MESS)"
+.TH MESS 6 2014-10-15 0.155 "The Multiple Emulator Super System (MESS)"
 .\"
 .\"
 .\" NAME chapter
@@ -935,8 +935,8 @@
 .B \-natural, \-nat
 Allows user to specify whether or not to use a natural keyboard.
 This allows you to start your game or system in a 'native' mode, depending
-on your region, allowing compatability for non\-"QWERTY" style keyboards.
-The default is OFF (-nonatural).
+on your region, allowing compatibility for non\-"QWERTY" style keyboards.
+The default is OFF (\-nonatural).
 .TP
 .B \-uimodekey, \-umk
 Specifies the key used to toggle between full and partial UI mode.
Index: mame/src/osd/sdl/man/romcmp.1
===================================================================
--- mame.orig/src/osd/sdl/man/romcmp.1	2014-08-15 05:24:23.000000000 +0200
+++ mame/src/osd/sdl/man/romcmp.1	2014-10-16 15:21:12.331521000 +0200
@@ -9,7 +9,7 @@
 .\" References
 .\" http://www.mame.net/mamefaq.html
 .\"
-.TH ROMCMP 1 2014-07-31 0.154 "MAME romset checking tool"
+.TH ROMCMP 1 2014-10-15 0.155 "MAME romset checking tool"
 .\"
 .\" NAME chapter
 .SH NAME
Index: mame/src/osd/sdl/man/testkeys.1
===================================================================
--- mame.orig/src/osd/sdl/man/testkeys.1	2014-08-15 05:24:23.000000000 +0200
+++ mame/src/osd/sdl/man/testkeys.1	2014-10-16 15:21:53.163521000 +0200
@@ -5,7 +5,7 @@
 .\" Man page created from source and usage information
 .\" Cesare Falco <c.falco@ubuntu.com>, February 2007
 .\"
-.TH TESTKEYS 1 2014-07-31 0.154 "MAME SDL keycode scanner"
+.TH TESTKEYS 1 2014-10-15 0.155 "MAME SDL keycode scanner"
 .\"
 .\" NAME chapter
 .SH NAME
Index: mame/src/osd/sdl/man/floptool.1
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ mame/src/osd/sdl/man/floptool.1	2014-08-12 19:53:30.000000000 +0200
@@ -0,0 +1,47 @@
+.\"  -*- nroff -*-
+.\"
+.\" floptool.1
+.\"
+.\" Man page created from usage info
+.\" Cesare Falco <c.falco@ubuntu.com>, April 2014
+.\"
+.\"
+.TH FLOPTOOL 1 2014-10-15 0.155 "MESS Generic floppy manipulation tool"
+.\"
+.\"
+.\" NAME chapter
+.SH NAME
+floptool \- MESS Generic floppy image manipulation tool
+.\"
+.\"
+.\" SYNOPSIS chapter
+.SH SYNOPSIS
+.B floptool
+.I option
+.\"
+.\"
+.\" DESCRIPTION chapter
+.SH DESCRIPTION
+.B floptool
+is a tool for the maintenance and manipulation of floppy disk
+images that MESS users need to deal with.
+.\"
+.\"
+.\" COMMANDS chapter
+.SH COMMANDS
+.TP
+.B identify \fIinputfile\fR [\fIinputfile \fR... ]
+Determine the format of a floppy disk image.
+.TP
+.B convert [\fIinputformat\fR|\fIauto\fR] \fIoutputformat inputfile outputfile
+Convert a floppy disk image to a different format.
+.\"
+.\"
+.\" EXAMPLES chapter
+.SH EXAMPLES
+.B floptool identify image.dsk
+.\"
+.\"
+.\" SEE ALSO chapter
+.SH SEE ALSO
+mess(6)
